.gitignore

```diff
@@ -16,9 +16,11 @@ site_kbtu_pct_change*.png
 
 # Ignore the out.osw files that have a custom tag (something like '_whatever' between 'out' and '.osw')
 test/*out_*.osw
+test/*out_*.status
```

Comment:
Probably ok to do these tests here but I didn't know if you were aware of the tests here: https://github.com/NREL/OpenStudio/tree/develop/openstudiocore/ruby/test and here https://github.com/NREL/OpenStudio/tree/develop/openstudiocore/src/cli/test ?

Especially this one: https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/cli/test/test_encodings.rb

Reply:
I think the unit tests in OpenStudio could be improved by calling them with both the CLI *and* the system ruby with -I pointing to the built bindings (not done currently):

https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/cli/CMakeLists.txt#L330

Reply:
Looks like we aren't calling the tests in /openstudiocore/ruby/test anyone, we should re-enable those.  They should be called with both the CLI and system ruby:

https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/cli/CMakeLists.txt#L330

Reply:
I wonder if these issues are related to https://github.com/NREL/openstudio-extension-gem/issues/12

Reply:
I wasn't aware of these tests in OpenStudio itself now. Do you want me to move stuff over there?
I guess the only thing that would make it more interesting to do it here is that we could keep track of a history of encodings (which I record, and aim to do so **by platform**) and success/failures.

Reply:
I don't think we can test this stuff enough, so I would be in favor of duplicate tests in both places.